adoc/Private-Registry-Installation.adoc

```diff
@@ -600,6 +600,10 @@ spec:
     image: registry.suse.com/harbor/harbor-redis:2.1.0
     customConfig:
```

Comment:
This `customConfig` section isn't really needed, could you please remove it in this PR ? The default dir value used by the operator is already `/data`.

Reply:
Nevermind, I was wrong, the redis operator does indeed rely on the default `./` setting and you need this set explicitly for our images.